lib/game/game.dart

```diff
@@ -87,6 +89,19 @@ abstract class Game {
     images.clearCache();
   }
 
+  /// Utility method to load and cache the image for a sprite based on its options
+  Future<Sprite> loadSprite(String path, SpriteOpts opts) async {
+    final image = await images.load(path);
+    return Sprite(image, opts);
+  }
+
+  /// Utility method to load and cache the image for a sprite animation based on its options
+  Future<SpriteAnimation> loadSpriteAnimation(
+      String path, SpriteAnimationOpts opts) async {
+    final image = await images.load(path);
+    return SpriteAnimation.fromImage(image, opts);
+  }
```

Comment:
Perhaps we could have this method in `Images` rather than here since there is nothing here that requires games properties?

Reply:
it requires the `images` property from game. I think it is better placed here since `Sprite` and `SpriteAnimations` are things that uses an `Image`. And I think it is nice to "center" this on the game, because we can add extension methods from our underlying packages, like a `loadSvg` from `flame_svg`.

Reply:
> it requires the images property from game

Let me rephrase myself: it does require anything from game that wouldn't be accessible in a `Images` instance. 

>  "center" this on the game, because we can add extension methods from our underlying packages, like a loadSvg from flame_svg

M only concern here is not to end up with a sueprclass.